docs/src/HowToGuides/Atmos/Microphysics.md

```diff
@@ -1,125 +1,277 @@
-# Microphysics Module
+# Microphysics
 
-The `Microphysics` module describes warm rain bulk parameterization of cloud microphysical processes. The module describes the warm rain (no ice and snow) formation and is based on the ideas of Kessler 1995.
-
-Parameterized processes include:
-  - rain sedimentation with mass weighted average terminal velocity,
-  - condensation/evaporation of cloud water,
-  - autoconversion,
-  - accretion,
-  - evaporation of rain water.
+The `Microphysics.jl` module describes a 1-moment bulk parameterization of cloud microphysical processes.
+The module is based on the ideas of
+[Kessler\_1995](https://www.sciencedirect.com/science/article/pii/016980959400090Z),
+[Grabowski\_1998](https://journals.ametsoc.org/doi/full/10.1175/1520-0469%281998%29055%3C3283%3ATCRMOL%3E2.0.CO%3B2 )
+and [Kaul\_et\_al\_2015](https://journals.ametsoc.org/doi/10.1175/MWR-D-14-00319.1)
 
 The cloud microphysics variables are expressed as specific humidities:
-  - q_tot - total water specific humidity,
-  - q_vap - water vapor specific humidity,
-  - q_liq - liquid water specific humidity,
-  - q_rai - rain water specific humidity.
-
-Parameters used in the parameterization are defined in `MicrophysicsParameters` module. They consist of:
+  - `q_tot` - total water specific humidity,
+  - `q_vap` - water vapor specific humidity,
+  - `q_liq` - cloud water specific humidity,
+  - `q_ice` - cloud ice specific humidity,
+  - `q_rai` - rain specific humidity,
+  - `q_sno` - snow specific humidity.
+
+## Assumed particle size relationships
+
+Particles are assumed to follow the mass(radius), cross section(radius), terminal velocity(radius)
+relationships defined as power laws.
+The coefficients are defined in `MicrophysicsParameters` module and are shown in the Table below.
```

Comment:
Aren't they part of CLIMAParameters? If so, might be good to mention.

Reply:
done